nocrypto.opam

```diff
@@ -0,0 +1,55 @@
+opam-version: "2.0"
+homepage:     "https://github.com/mirleft/ocaml-nocrypto"
+dev-repo:     "git+https://github.com/mirleft/ocaml-nocrypto.git"
+bug-reports:  "https://github.com/mirleft/ocaml-nocrypto/issues"
+doc:          "https://mirleft.github.io/ocaml-nocrypto/doc"
+authors:      ["David Kaloper <david@numm.org>"]
+maintainer:   "David Kaloper <david@numm.org>"
+license:      "ISC"
+tags:          [ "org:mirage" ]
+
+build: [
+  ["ocaml" "pkg/pkg.ml" "build" "--pkg-name" "%{name}%"
+           "--pinned" "%{pinned}%" "--tests" "false"
+           "--with-lwt" "%{lwt+cstruct-lwt:installed}%"
+           "--xen" "%{mirage-xen-ocaml:installed}%"
+           "--mirage" "%{mirage-entropy+lwt:installed}%"
+           "--freestanding" "%{ocaml-freestanding:installed}%"]
+  [ "ocaml" "pkg/pkg.ml" "build" "--pkg-name" "%{name}%"
+            "--pinned" "%{pinned}%" "--tests" "true" ] {with-test}
+  [ "ocaml" "pkg/pkg.ml" "test" "--pkg-name" "%{name}%"] {with-test}
+]
+
+depends: [
+  "ocaml" {>= "4.02.0"}
+  "ocamlfind" {build}
+  "ocamlbuild" {build}
+  "topkg" {build}
+  "cpuid" {build}
```

Comment:
also not just a build dependency - you probably want to recompile if the cpuid library changes